Dockerfile

```diff
@@ -54,8 +54,6 @@ RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get -y --no-install-r
 #########################
 ## Python Dependencies ##
 #########################
-COPY mssql-requirements.txt .
-RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then pip --no-cache-dir install -r mssql-requirements.txt ; fi
```

Comment:
@sanders41 do you know if this breaks straight-up if the driver hasn't already been installed? I recall this being a specifically M1 issue so I don't think I can test it properly

This has changed because we're now using proper optional dependencies and I want to make sure I don't regress on anything here

Reply:
I did a build with the driver install parts removed and didn't get any errors. 

But if I try to install optional requirements locally without docker I do get errors on the MSSQL install. This will be problematic for editors during development.

Reply:
@sanders41 that seems like expected behavior, how did you try to install the optional requirements?

Reply:
`pip install -r optinal-requirements.txt`. If I comment out `pyodbc` (the mssql driver) it installs without issue.

Reply:
working as intended then, right? M1 Macs can't install that package. The solution is to use the optional dependencies as designed instead of directly using the requirements file, so something like `pip install .[all]` or in the real scenario `pip install ethyca-fides[bigquery]`

Reply:
Yeah, `pip install -e .[extras]` would work for development on a mac since it skips mssql.

Reply:
That will affect this part though right?

```Dockerfile
# SQL Server (MS SQL)
# https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15
RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get install -y --no-install-recommends apt-transport-https && apt-get clean && rm -rf /var/lib/apt/lists/* ; fi
RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - ; fi
RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then curl https://packages.microsoft.com/config/debian/11/prod.list | tee /etc/apt/sources.list.d/msprod.list ; fi
RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get update ; fi
ENV ACCEPT_EULA=y DEBIAN_FRONTEND=noninteractive
RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get -y --no-install-recommends install \
    unixodbc-dev \
    msodbcsql17 \
    mssql-tools \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* ; fi
```

This can skip installing MSSQL stuff but `optional-requirements.txt` here will install it. I didn't run the test suite, but just installing I could remove all of these lines and install without error so I don't know if they are still needed or not.

Also the `SKIP_MSSQL_INSTALLATION` arg doesn't seem to do anything any more since `pip install optional-requirements.txt` will install the driver no matter what is set for this arg.

Reply:
good point, that's kind of a moot point now.

Is there any reason not to completely remove the `MSSQL` arg stuff? We never used it on ctl and never had issues

Reply:
Based on my build test I think it is safe to remove.